server/src/main/java/org/opensearch/index/codec/CodecService.java

```diff
@@ -91,6 +94,17 @@ public Codec codec(String name) {
         return codec;
     }
 
+    public Codec codec(String name, int compressionLevel) {
```

Comment:
Do we need this method? We can init with the required level in the constructor itself

Reply:
yeah, it's not required. I've updated this